tools/tertiary-analysis/scanpy/scanpy-read-10x.xml

```diff
@@ -42,6 +42,10 @@ scanpy-read-10x.py
   </tests>
 
   <help><![CDATA[
+==========================================================
+Read 10x-Genomics-formatted mtx directory (`read_10x_mtx`)
+==========================================================
+
```

Comment:
Shouldn't we succinctly describe what are the expected format of the files to be given as input (ie. Matrix Market, genes as columns, cells as rows, etc)? Maybe also mention that this can read from our Atlas SC Downloader tool?

Reply:
Thank! Description added.